apps/front/src/components/atoms/Alert.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { InputHTMLAttributes } from 'react';
+
+interface AlertProps {
+    alertType?: 'alert' | 'alert-success' | 'alert-error' | 'alert-warning' | 'alert-info';
+    label?: string;
+    // daisyUIのbtnプロパティ
+    btnType?: 'btn-success' | 'btn-error' | 'btn-warning' | 'btn-info';
+    btnLabel?: string;
+    btnDisplay?: boolean;
+};
+
+/**
+ * daisyUI使用のAlert
+ * 
+ * @example
+ * export default function Home() {
+ *     return (
+ *             <Alert  type='alert-error' label='testText' btnType="btn-success" btnLabel="testButton" btnDisplay={false}> 
+ *     );
+ * }
+ */
+
+export const Alert: React.FC<AlertProps> = ({
+    alertType,
+    label,
+    btnType,
+    btnLabel,
+    btnDisplay = false,
```

Comment:
```suggestion
    btnDisplay = false,
    onClick
```
アラート内のボタン押した時のonClickを外から呼び出せるようにしておいてほしい！！

Reply:
レビューありがとう！追加しました🙏